legacy/emergency-freeze/20250604_084529/continuity-states/mcp-continuity-service/src/cli.py

```
@@ -0,0 +1,96 @@
+            if result.get('projects'):
+                console.print(Panel(f"Projects: {len(result['projects'])}", title="Projects"))
+                
+            if result.get('critical_missions'):
+                console.print(Panel(f"Critical Missions: {len(result['critical_missions'])}", title="Missions"))
+                
+            if result.get('next_actions'):
+                console.print(Panel(str(result['next_actions']), title="Next Actions"))
+    
+    except Exception as e:
+        console.print(f"❌ Error: {e}")
+        raise click.ClickException(str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)